packages/kbn-securitysolution-utils/src/path_validations/index.test.ts

```diff
@@ -65,32 +84,55 @@ describe('validateFilePathInput', () => {
         : OperatingSystem.LINUX;
 
     it('does not warn on valid filenames', () => {
-      expect(validateFilePathInput({ os, value: '/opt/*/FILENAME.EXE-1231205124.gz' })).not.toEqual(
-        FILENAME_WILDCARD_WARNING
-      );
       expect(
-        validateFilePathInput({
+        validateWildcardInput({
+          field: 'file.path.text',
+          os,
+          value: '/opt/*/FILENAME.EXE-1231205124.gz',
+        })
+      ).not.toEqual(WILDCARD_WARNING);
+      expect(
+        validateWildcardInput({
+          field: 'file.path.text',
           os,
           value: "/opt/*/test$  as2@13---12!@#A,DS.#$^&$!#~ 'as'd.华语.txt",
         })
-      ).not.toEqual(FILENAME_WILDCARD_WARNING);
+      ).not.toEqual(WILDCARD_WARNING);
     });
     it('warns on wildcard in file name at the end of the path', () => {
-      expect(validateFilePathInput({ os, value: '/opt/bin*' })).toEqual(FILENAME_WILDCARD_WARNING);
-      expect(validateFilePathInput({ os, value: '/opt/FILENAME.EXE-*.gz' })).toEqual(
-        FILENAME_WILDCARD_WARNING
+      expect(validateWildcardInput({ field: 'file.path.text', os, value: '/opt/bin*' })).toEqual(
+        WILDCARD_WARNING
+      );
+      expect(
+        validateWildcardInput({ field: 'file.path.text', os, value: '/opt/FILENAME.EXE-*.gz' })
+      ).toEqual(WILDCARD_WARNING);
+    });
```

Comment:
I would suggest not to not change these tests but write new ones for `validateWildcardInput` function. Also, for file paths that have a wildcard I should see the file path specific warning.